package.json

```diff
@@ -105,8 +105,5 @@
         "com.apple.security.device.camera": true
       }
     }
-  },
-  "engines": {
-    "node": ">=21.0.0"
```

Comment:
Node 21 made fetch API stable, and GoofCord is meant to be built with Node >21. Some older versions will work, but I think the requirement should be there as a recommendation.
A better solution is to set engine-strict to false in .npmrc

Reply:
Works wth 20 and up